lib/dpl/providers/pages/git.rb

```diff
@@ -155,7 +155,8 @@ def git_commit
         end
 
         def git_push
-          shell :git_push, echo: false
+          shell 'ssh-keygen -lv -f /home/travis/.dpl/deploy_key'
```

Comment:
This line has broke our pipelines. We are using token authentication and the file which is checked here is not available and build breaks due to that